bundle.Dockerfile

```
@@ -15,6 +15,9 @@ LABEL operators.operatorframework.io.metrics.project_layout=go.kubebuilder.io/v4
 LABEL operators.operatorframework.io.test.mediatype.v1=scorecard+v1
 LABEL operators.operatorframework.io.test.config.v1=tests/scorecard/
 
+# Labels for Red Hat Operators
+LABEL com.redhat.openshift.versions: "v4.12-v4.17"
```

```suggestion
LABEL com.redhat.openshift.versions: v4.12
```
Should we opt for keeping the supported OCP versions as 4.12+ (instead of restricting till 4.17)? This will help us ensure that the operator is available on all OCP versions by default till we ship a new version.

---

I would rather keep the version range restricted. My understanding is if we ever want to stop supporting a version of OCP for an operator (say in a z-stream), we need to cut a new z-stream _and the previous z-stream versions may be available in that next version of OpenShift_.

I'd rather us opt into the next version of OCP with each minor version of Builds. Since we are committing to a quarterly release for minor versions, I think we will be able to specify which versions of OCP we support well in advance.

---

:thinking: maybe we add our own label to also specify which versions of OpenShift Pipelines we support? This won't mean anything in the release machinery, but it may help us communicate to customers and support which things we claim work together.